modules/machine-pools-hcp.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine-pools-hcp_{context}"]
+= Machine pools in {hcp-title} clusters
+
+In {hcp-title} clusters, the {hcp} spans all three of the availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Additionally, machine pools in {hcp-title} clusters each upgrade independently and the versions of the machine pools must remain within two minor (Y-stream) versions of the control plane. 
```

Comment:
all three... suggest mentioning as 
spans across three Availability Zones. 

Because some regions have > 3 AZs.

Reply:
_Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ._

Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. And, each AZ can only have one machine pool.

Reply:
_Additionally, machine pools in {hcp-title} clusters each upgrade independently and the versions of the machine pools must remain within two minor (Y-stream) versions of the control plane._

Here, we're not talking about upgrades of machine pool. That'll be covered in the Cluster Upgrade section. We should only mention about machine pool version here. 

When you create a machine pool, it will be at the same OpenShift x.y.z version as the HCP. However, when the HCP is upgraded, there is not need to upgrade the machine pools. Thus, over time, machine pool can remain within 2 Y-versions from HCP's version.